engine/gamesys/src/gamesys/components/comp_camera.cpp

```diff
@@ -179,6 +179,11 @@ namespace dmGameSystem
         return dmGameObject::CREATE_RESULT_OK;
     }
 
+    void* CompCameraGetComponent(const dmGameObject::ComponentGetParams& params)
+    {
+        return (void*)params.m_UserData;
+    }
```

Comment:
The camera already returned the pointer from the `CompCameraCreate()`function, so no conversion needed.
While I didn't need to add this function to the camera, I though it better to add it to the most common component types. We can later add for the rest of them as well, but I ran out of time. Also, the tests checks some that don't have the function (e.g. collectionproxy)